appveyor_mingw_cygwin.yml

```diff
@@ -32,9 +32,14 @@ install:
          mingw-w64-x86_64-gtest ; do (echo y | pacman -S $i) ; done
     - cd %APPVEYOR_BUILD_FOLDER%
     - if "%BUILD%"=="CYGWIN64" set "PATH=c:\cygwin64\usr\local\bin;c:\cygwin64\bin;c:\cygwin64\usr\bin;c:\cygwin64\usr\sbin;"
+    
     - if "%BUILD%"=="CYGWIN64" C:\cygwin64\bin\bash -c "wget https://raw.githubusercontent.com/transcode-open/apt-cyg/master/apt-cyg ; chmod +x apt-cyg; mv apt-cyg /usr/local/bin"
-    - if "%BUILD%"=="CYGWIN64" C:\cygwin64\bin\bash -c "apt-cyg install cmake zlib-devel libexpat-devel libxml2-devel libxslt-devel python38 python38-pip python38-libxml2"
+    # Use change mirror used by apt-cyg.  #2003 cmake installed by default mirror is broken 
+    - if "%BUILD%"=="CYGWIN64" C:\cygwin64\bin\bash -c "apt-cyg mirror http://mirrors.kernel.org/sources.redhat.com/cygwin/"
+    - if "%BUILD%"=="CYGWIN64" C:\cygwin64\bin\bash -c "apt-cyg install cmake curl zlib-devel libexpat-devel libxml2-devel libxslt-devel python38 python38-pip python38-libxml2 tar"
     - if "%BUILD%"=="CYGWIN64" C:\cygwin64\bin\bash -c "/usr/bin/python3.8.exe -m pip install --upgrade pip"
+    # As a last resort, build CMake from source.  Caution:  This takes about 60 minutes.
+    # if "%BUILD%"=="CYGWIN64" C:\cygwin64\bin\bash -c "cd /tmp; curl -LO https://github.com/Kitware/CMake/releases/download/v3.22.0/cmake-3.22.0.tar.gz; tar xfz cmake-3.22.0.tar.gz; cd cmake-3.22.0 ; ./bootstrap ; make ; make install" 
```

Comment:
Remove? Should not really be an option anyway...

Reply:
I think we should retain that comment.

If you want to polish the MinGW code, that's fine with me.

Reply:
Ok, will do on this branch in a bit, thanks.